# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../icc/pkg_gawk_memcpy.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./missing_d/memcpy.c.orig	Fri Jun 21 18:50:43 2002
+++ ./missing_d/memcpy.c	Fri Jun 21 18:51:02 2002
@@ -4,6 +4,7 @@
  * We supply this routine for those systems that aren't standard yet.
  */
 
+/*
 char *
 memcpy (dest, src, l)
 register char *dest, *src;
@@ -16,3 +17,4 @@
 
 	return ret;
 }
+*/
